recipes-android/android-init/android-init/android-init.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=/system/bin/init initializes the property service and launches servicemanager and logd
+After=local-fs.target
+Before=basic.target network.target bluetooth.service ofono.service sensord.service
+DefaultDependencies=no
+Conflicts=shutdown.target
+
+[Service]
+Type=simple
+ExecStartPre=/bin/touch /dev/.coldboot_done
+ExecStart=/system/bin/init --second-stage
```

Comment:
Would this be needed in other watches too ?

Reply:
Only sometimes. It wasn't needed for bass but appears to be required for sturgeon. It's copied from lenok so I would guess it's required there too.